### PR TITLE
fixes theming errors for M3 chips

### PR DESCRIPTION
**Pull request by banoodle**
## Description
Teamwork Ticket(s): N/A

This PR applies the changes Sean made to fix the M3 theming errors on irc-client.

## Steps to Validate
Is there a way to validate this without having to fully provision a new project?

We have unit-tested these changes on the Flagler and IRC-Client projects and they were based on an earlier version of drupal-starter, so I think these changes will work.

If you want me to do a full end-to-end test with test repo and pantheon account I will, but I worry that might be overkill.

## Affected URL
N/A

### CODE REVIEW

File: .docksal/docksal-local.yml.example [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: '3.9'
+services:
+  db:
+    image: docksal/mariadb:10.4
```
Comment (thejimbirch) [inline .docksal/docksal-local.yml.example:4]: Shouldn't this be `10.6`?

File: .docksal/docksal-local.yml.example [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    platform: linux/arm64
+  cli:
+    image: docksal/cli:php8.1-3
```
Comment (thejimbirch) [inline .docksal/docksal-local.yml.example:7]: @sean-e-dietrich Should this be upped to `8.2`?

File: .docksal/docksal-local.yml.example (hunk 1 above)
Comment (thejimbirch) [inline .docksal/docksal-local.yml.example:4]: Shouldn't this be `10.6`?

File: .docksal/commands/init [hunk 3]
```diff
@@ -96,6 +96,9 @@ fin reset -f
 echo "Waiting 10s for MySQL to initialize.";
 sleep 10
 
+# Symlink Drush
+fin exec '([ ! -f /usr/local/bin/drush ] || false) && sudo ln -sF /var/www/vendor/bin/drush /usr/local/bin/drush'
```
Comment (thejimbirch) [inline .docksal/commands/init:100]: @sean-e-dietrich I have the following on PCG.  What should it be, `true` or `false`?

`fin exec '([ -f /usr/local/bin/drush ] || true) && sudo ln -sf /var/www/vendor/bin/drush /usr/local/bin/drush'`

**Review by thejimbirch — COMMENTED** [review]
@sean-e-dietrich I have a couple of questions for you on this one.

**Comment by paulsheldrake** [comment]
I think we should close this PR and focus on making sure projects that are using storybook get upgraded.   adding more containers to the starter isn't the long term fix for this

**Comment by paulsheldrake** [comment]
There are notes in the slack comment on how to update KDCL so we don't have to add more containers to the starter
https://kanopi.slack.com/archives/C013LA8DRFU/p1710808935443249

**Comment by paulsheldrake** [comment]
@thejimbirch What do you think ⬆️

**Comment by thejimbirch** [comment]
I think we need to spin up a couple of PRs against active builds, Flagler, IRC, Parks to verify.

**Comment by thejimbirch** [comment]
We have verified on a few projects. Closing.

**Pull request closed without merging**